tests/scripts/all.sh

```diff
@@ -1662,6 +1672,7 @@ component_test_valgrind_constant_flow_psa () {
     msg "build: cmake release GCC, full config with constant flow testing"
     scripts/config.py full
     scripts/config.py set MBEDTLS_TEST_CONSTANT_FLOW_VALGRIND
+    skip_non_constant_flow_components
```

Comment:
Uh-oh, we don't have a `test_valgrind_psa`. So there's loss of coverage here. We should either keep this component as is or add a `test_valgrind_psa`.